### PR TITLE
improve validation

### PR DESCRIPTION
**Pull request by VolodymyrKaranevych**

### CODE REVIEW

File: src/components/Dashboard/MainInput/MainInput.jsx [hunk 1]
```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
     });
   };
 
-  setInputValue = (value) => {
+  setInputValue = (value = '') => {
     const { newPriority } = this.state;
-    const inputDataArr = value.split('');
+    const inputDataArr = value;
```
Comment (opanki) [inline src/components/Dashboard/MainInput/MainInput.jsx]: inputDataArr const is not needed any more

File: src/components/Dashboard/MainInput/MainInput.jsx [hunk 2]
```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
     });
   };
 
-  setInputValue = (value) => {
+  setInputValue = (value = '') => {
```
Comment (opanki) [inline src/components/Dashboard/MainInput/MainInput.jsx]: you can skip brackets if function takes only one param

File: src/components/Dashboard/MainInput/MainInput.jsx [hunk 3]
```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
...
     const { newPriority } = this.state;
-    const inputDataArr = value.split('');
+    const inputDataArr = value;
     const hashCheck = inputDataArr[inputDataArr.length - 2];
     let setPriority = newPriority;
```
Comment (opanki) [inline src/components/Dashboard/MainInput/MainInput.jsx]: will be overrided inside changePriority ?

File: src/components/Dashboard/MainInput/MainInput.jsx [hunk 4]
```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
...
     if (hashCheck === '#') {
       const getPriority = inputDataArr[inputDataArr.length - 1];
-      if (getPriority === '1') {
-        setPriority = 'High';
-      } else if (getPriority === '3') {
-        setPriority = 'Low';
-      } else {
-        setPriority = 'Medium';
-      }
+      setPriority = changePriority(getPriority);
```
Comment (opanki) [inline src/components/Dashboard/MainInput/MainInput.jsx]: not really understandable variable/func names, I would suggest something like
```
getPriority => enteredPriority/priorityIndex
changePriority => getPriority
setPriority  => newPriority
```

File: src/components/Dashboard/MainInput/MainInput.jsx (hunk 2 above)
Comment (mishalisovskiy) [reply src/components/Dashboard/MainInput/MainInput.jsx]: If I may, I would like to disagree here. According to the style guide we are using for this project (AirBnB rules), parameter brackets may be skipped only if a function does not have a curly braces body.

Comment (opanki) [reply src/components/Dashboard/MainInput/MainInput.jsx]: good point, no problem from my side

File: src/components/Dashboard/MainInput/MainInput.jsx (hunk 3 above)
Comment (VolodymyrKaranevych) [reply src/components/Dashboard/MainInput/MainInput.jsx]: Yes, it was stupid. I changed it.

File: src/components/Dashboard/MainInput/MainInput.jsx (hunk 4 above)
Comment (VolodymyrKaranevych) [reply src/components/Dashboard/MainInput/MainInput.jsx]: It was changed.

File: src/components/Dashboard/MainInput/MainInput.jsx (hunk 1 above)
Comment (VolodymyrKaranevych) [reply src/components/Dashboard/MainInput/MainInput.jsx]: It was deleted.

**Pull request merged**